htdocs/resources/views/master_layout.blade.php

```diff
@@ -80,10 +79,8 @@
 </div>
 
 
-<!-- jQuery (necessary for Bootstrap's JavaScript plugins) -->
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
-<!-- Include all compiled plugins (below), or include individual files as needed -->
-<script src="http://{{ $server_host }}/js/bootstrap.min.js"></script>
-@yield('typeahead')
+<!-- Load All JS from a single concatenated file, see gulpfile.js and resources/assets/js/app.js -->
+<script src="{{ elixir("js/app.js") }}"></script>
```

Comment:
all JS is bundled in this file. This file is versioned so it is cash busted every time I build a new one